README.md

```diff
@@ -54,6 +54,17 @@ Then you can run the scripts in the correct order marked in each of the scripts
 node x.<script>
 ```
 
+**WARNING**
+
+This is valid only for the Alastria environment. If we work with an internal implementation for testing purposes, we will have to change some parameters in the configuration files:
+
+- Smartcontracts deployed addresses: found in `node_modules/alastria-identity-lib/dist/config.js`
```

Comment:
```suggestion
- Smart Contracts deployed addresses: found in `node_modules/alastria-identity-lib/dist/config.js`
```

Reply:
Indeed, if you want to change the SC addresses, you need to:
1. Add the new addresses in `node_modules/alastria-identity-lib/alastria-identity/contracts/ContractInfo.md`
2. Then execute the script `configFile.js`. This can be done running `node src/configFile.js` when you are in the directory  `node_modules/alastria-identity-lib`
3. Finally transpilate with `tsc`